raft/handler.go

```diff
@@ -487,6 +487,10 @@ func (pm *ProtocolManager) startRaft() {
 			for _, entry := range entries {
 				if entry.Type == raftpb.EntryNormal {
 					var block types.Block
+					if len(entry.Data) == 0 {
```

Comment:
Is this related to QBFT? Should this be covered as part of a separated PR?

Reply:
This should no longer be needed after the `DecodeRLP()` changes are reverted. I have pushed a commit to remove this